lib/screens/register_screen.dart

```diff
@@ -17,181 +20,272 @@ class RegisterScreen extends StatefulWidget {
+                                      ),
+                                      SizedBox(width: 5.0),
+                                      Text(
+                                        '!',
+                                        style: TextStyle(
+                                          fontSize: 65.0,
+                                          fontWeight: FontWeight.bold,
+                                          color: kPrimaryColor,
+                                        ),
+                                      ),
+                                    ],
+                                  ),
+                                ),
+                              ],
                             ),
-                            Padding(
-                              padding: const EdgeInsets.only(top: 50.0),
-                              child: Row(
-                                crossAxisAlignment: CrossAxisAlignment.baseline,
-                                textBaseline: TextBaseline.alphabetic,
+                          ),
+                          SizedBox(height: 20.0),
+                          CustomTextFeild(
+                            label: 'NAME',
+                            prefixIcon: Icon(Icons.person),
+                            textCap: TextCapitalization.words,
+                            changed: (value) {
+                              this._name = value.trim();
+                            },
+                          ),
+                          SizedBox(height: 10.0),
+                          CustomTextFeild(
+                            label: 'EMAIL',
+                            prefixIcon: Icon(Icons.email),
+                            kbType: TextInputType.emailAddress,
+                            changed: (value) {
+                              this._email = value.trim();
+                            },
+                          ),
+                          SizedBox(height: 10.0),
+                          CustomTextFeild(
+                            label: 'PASSWORD',
+                            prefixIcon: Icon(Icons.lock),
+                            isPass: true,
+                            changed: (value) {
+                              this._password = value.trim();
+                            },
+                          ),
+                          SizedBox(height: 10.0),
+                          CustomTextFeild(
+                            label: 'CONFIRM PASSWORD',
+                            prefixIcon: Icon(Icons.lock),
+                            isPass: true,
+                            changed: (value) {
+                              this._confirmPassword = value.trim();
+                            },
+                          ),
+                          SizedBox(height: 20.0),
+                          Consumer<AddressModel>(
+                            builder: (context, addressModel, child) {
+                              return Row(
```

Comment:
Size of content in this row should be dynamic.

Reply:
Done.